demo.sh

```
@@ -0,0 +1,137 @@
+[[ -d "/tmp/ralph-demo-install/sessions" ]] && check "install.sh creates sessions/" 0 || check "install.sh creates sessions/" 1
+
+# Test installed version works
+INSTALLED_VERSION=$(/tmp/ralph-demo-install/ralph.sh --version 2>&1)
+[[ "$INSTALLED_VERSION" == *"ralph"* ]] && check "installed ralph.sh --version works" 0 || check "installed ralph.sh --version works" 1
+
+# Cleanup
+rm -rf /tmp/ralph-demo-install
+echo ""
+
+echo -e "${BOLD}5. Testing pretty-printer${NC}"
+echo -e "${DIM}   Command: echo '{"type":"text","text":"Hello"}' | ./ralph-pretty-print.sh${NC}"
```

Shellcheck (0.11.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)